build-windows/build.sh

```diff
@@ -73,6 +85,18 @@ if [ "${MONO}" == "1" ]; then
   mkdir -p /root/out/x86_32/templates-mono
   cp -rvp bin/* /root/out/x86_32/templates-mono
   rm -rf bin
+
+  $SCONS platform=windows arch=arm64 $OPTIONS $OPTIONS_MONO $OPTIONS_LLVM target=editor
+  ./modules/mono/build_scripts/build_assemblies.py --godot-output-dir=./bin --godot-platform=windows
```

Comment:
Unrelated to the PR, but do we need to build .NET assemblies for each architecture, it seems to contain only managed code, and scripts aren't even taking `arch` as argument, so it probably can be done one.